src/main/java/appeng/tile/storage/TileDrive.java

```diff
@@ -85,30 +102,19 @@ public TileDrive()
 	@TileEvent( TileEventType.NETWORK_WRITE )
 	public void writeToStream_TileDrive( final ByteBuf data )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00100)